plugins/modules/get_certificate.py

```diff
@@ -325,6 +336,10 @@ def main():
             if start_tls_server_type is not None:
```

Comment:
Please fail in line ~302 when `ciphers` is set.

Reply:
A little confused here by the recommended line number. Do you mean something like this?
```
    if not HAS_CREATE_DEFAULT_CONTEXT:
        # Python < 2.7.9
        if proxy_host:
            module.fail_json(msg='To use proxy_host, you must run the get_certificate module with Python 2.7 or newer.',
                             exception=CREATE_DEFAULT_CONTEXT_IMP_ERR)
        if ciphers:
            module.fail_json(msg='To use ciphers, you must run the get_certificate module with Python 2.7 or newer.', exception=CREATE_DEFAULT_CONTEXT_IMP_ERR)

        try:
            # Note: get_server_certificate does not support SNI!
            cert = get_server_certificate((host, port), ca_certs=ca_cert)
        except Exception as e:
            module.fail_json(msg="Failed to get cert from {0}:{1}, error: {2}".format(host, port, e))
    else:
```

Reply:
Unfortunately in GitHub it isn't possible to add annotations to lines that aren't close to modified ones. (That's one of the things GitLab does better IMO :) )